.bumpversion.cfg

```diff
@@ -1,28 +1,28 @@
 [bumpversion]
 name = Rx
 project = RxPY
-current_version = 3.0.0-beta4
+current_version = 3.0.0b4
```

Comment:
doesn't this break the 'release' scheme ?

Reply:
I think it needs to match the release values!? Anyways, the when uploaded to pypi it got version v3, even if it was beta3 here. What was the problem?

Reply:
Well, setup.py was printing warnings about “normalizing” the version string. And on pypi the value does end up as “3.0.0b4”: https://pypi.org/project/Rx/#history

I actually agree the original version string is nicer, I just have a thing about wanting to get rid of warnings, I suppose.

Also I now realize I haven’t checked if the bumpconfig tool understands this shorter variant, although if it’s apparently the standard way I would assume it does.

Anyway, naturally I’ll change it back if you want!?

Reply:
By the way, personally I find I slightly dislike having the project metadata in a file named “bumpversion.cfg” and needing to install a separate tool to edit a value in two places... But there we are.

Reply:
I agree. We should just remove it. It makes more problems than it solves.

Reply:
I've removed the commit about the version string here, let's address it separately (or not).

> We should just remove it.

You mean the tool? And if we remove it, do you think we should have the "full" version string or the shorter one (I think it's not bumpversion which does this normalizing, but setuptools).

Reply:
Yes, let's remove bumpversion. We have a tool to update a value in two places, and if we remove the tool then we only need to update it in one place? Do we need a tool to do that. I thought it would be simpler with the tool, but it made releasing a new beta so much more complicated.

I'm btw totally agnostic about the format of the version string.

Reply:
No, we’d still need to edit in two places. I just think a tool for only that purpose is overkill. I’ll make a proposal when I get to campus.

Reply:
Ok, sorry that this did not meet the goal. On my projects it makes me save time so I thought it would help here. Obviously if this is more a pain than a benefit, let's remove it. The original issue was to avoid reading the config file at runtime, which is still solved with the new proposal of Erik.

Reply:
Ok, sorry that this did not meet the goal. On my projects it makes me save time so I thought it would help here. Obviously if this is more a pain than a benefit, let's remove it. The original issue was to avoid reading the config file at runtime, which is still solved with the new proposal of Erik.

Reply:
Actually it did meet the goal, I guess, but as I said my view is perhaps it is a bit much to have a separate tool for editing two strings? And it left important metadata in a file whose name did not convey its purpose (and was hidden). If there’s some benefit to keeping bumpversion that I am overlooking, then please let me know?